### DISCUSSION TITLE
Response time too high

### DISCUSSION THREAD

**Discussion by pdns90** [deepgram/community#556] [General help] [answered]
Hi there, Our audio file is in HINDI like 2-3 minutes and file size around 6 MB and if we use whisper model like this https://api.deepgram.com/v1/listen?smart_format=true&language=en&model=whisper so response time is around 2 minutes.

Also, if we use ENGLISH audio and use general model (https://api.deepgram.com/v1/listen?model=general)  then same file size around 6 MB then we are getting response 5-6 seconds so all is fine with English audio.

Please let me know which model is good for HINDI audio so we can get fast response.

Also, Our server bandwidth is very good.

Please help on this.

Thanks,

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment]
Hi @pdns90, we support our latest Nova-2 model for Hindi, so I recommend using that. `model=nova-2&language=hi` (or if you prefer to receive a transcript with Latin characters, we have `language=hi-Latn`). You can see our full [Models and Languages documentation](https://developers.deepgram.com/docs/models-languages-overview).

**Comment by pdns90** [comment]
Hi there, thanks for your message.

We are giving interview in HINDI but we need transcript in ENGLISH, as per above your message we are getting blank output.

Please help.

Thanks

**Reply by jkroll-deepgram** [reply, marked as answer]
Hi @pdns90, when transcribing English audio with our general model, that is Deepgram's own model. When translating Hindi to English, that is our managed Whisper model. Whisper is significantly slower than Deepgram's own models, plus the translation adds an extra computational step. 

One option that can reduce the latency is to use `model=whisper-small`, where our parameter`model=whisper` defaults to `whisper-large`.

I would expect to see comparable latency for transcribing both English and Hindi audio using Whisper.
